resources/views/questions/single.blade.php

```diff
@@ -3,8 +3,8 @@
         ? "survey::questions.types.{$question->type}" 
         : "survey::questions.types.text",[
             'disabled' => !($eligible ?? true), 
-            'value' => $lastEntry ? $lastEntry->answerFor($question) : null,
+            'value' => (($lastEntry ?? null) !== null) ? $lastEntry->answerFor($question) : null,
             'includeResults' => ($lastEntry ?? null) !== null
         ]
     )
-</div>
\ No newline at end of file
+</div>
```

Comment:
@stojankukrika thanks for your contribution. Was this closing `div` meant to be removed?